js/main.js

```diff
@@ -0,0 +1,43 @@
+var $buttonList = document.querySelector('.button-list');
+var $homePage = document.querySelector('.homepage');
+var $menuIcon = document.querySelector('.fas.fa-bars.hidden');
+var $placeHolder = document.querySelector('.place-holder');
+var $tailWags = document.querySelector('.tailwags');
+var $intro = document.querySelector('.full-col.intro');
+
+var names = ['Bella', 'Luna', 'Charlie', 'Lucy', 'Cooper', 'Max', 'Bailey', 'Daisy', 'Sadie', 'Maggie', 'Rocco', 'Lucky', 'Dozer'];
+
+var randomName = names[Math.floor(Math.random() * names.length)];
+
+var sayHi = ['Hi, ', 'Nice to meet you, ', 'Hello, ', 'Hey there, '];
+var randomGreeting = sayHi[Math.floor(Math.random() * sayHi.length)];
```

Comment:
You should make this into a function so you can repeat this functionality for new searches